elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1768,7 +1768,7 @@ protected void auditField(final ChangeSpec changeSpec) {
         }
         for (Audit annotation : annotations) {
             if (annotation.action().length == 1 && annotation.action()[0] == Audit.Action.UPDATE) {
-                LogMessage message = new LogMessage(annotation, this, Optional.of(changeSpec));
+                LogMessageImpl message = new LogMessageImpl(annotation, this, Optional.of(changeSpec));
```

Comment:
```suggestion
                LogMessage message = new LogMessageImpl(annotation, this, Optional.of(changeSpec));
```

Reply:
Somewhat out of scope, could this object creation be injected or provided by a factory class?

Reply:
👍